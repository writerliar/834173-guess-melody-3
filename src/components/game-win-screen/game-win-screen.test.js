import React from "react";
import renderer from "react-test-renderer";
import GameWinScreen from "./game-win-screen.jsx";

it(`Should GameWinScreen render correctly`, () => {
  const tree = renderer.create(
      <GameWinScreen questionsCount={5} mistakesCount={2} onReplayButtonClick={()=>{}}/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
