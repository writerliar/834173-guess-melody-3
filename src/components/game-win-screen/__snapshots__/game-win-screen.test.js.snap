// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GameWinScreen render correctly 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="result__total"
  >
    Вы ответили правильно на $
    3
     вопросов и совершили $
    2
     ошибки
  </p>
  <button
    className="replay"
    onClick={[Function]}
    type="button"
  >
    Сыграть ещё раз
  </button>
</section>
`;
